mlos_bench/mlos_bench/optimizers/convert_configspace.py

```diff
@@ -7,7 +7,8 @@
 """
 
 import logging
-from typing import Dict, Hashable, List, Optional, Tuple, Union
+from collections.abc import Hashable
+from typing import Dict, List, Optional, Tuple, Union
```

Comment:
Let's split this out to another PR too - also unrelated change in the linter rules.

Reply:
This is not a change in linter rules -- Hashable was depreciated